app/src/main/res/values/colors.xml

```diff
@@ -50,6 +50,14 @@
     <color name="snackbarTextColor">#ffffffff</color>
     <color name="snackbarActionText">@color/photonBlue50</color>
 
+    <color name="searchSuggestionsBackground">@color/photonInk80</color>
+
+    <color name="searchSuggestionPromptBackground">@color/photonInk80</color>
+    <color name="searchSuggestionPromptTitleColor">@color/photonGrey10</color>
+    <color name="searchSuggestionPromptSubtitleColor">#99f9f9fa</color>
+    <color name="searchSuggestionPromptBorderColor">#19f9f9fa</color>
```

Comment:
If those transparent are drawn over each other to create a **solid** color then it's usually way cheaper to just draw the "combined" color. Is this the case here?

* https://developer.android.com/topic/performance/rendering/overdraw#rt
* https://www.youtube.com/watch?v=wIy8g8yNhNk&index=46&list=PLWz5rJ2EKKc9CBxr3BVjPTPoDPLdPIFCE